### ISSUE TITLE
Provide key figures of task in result header

### ISSUE THREAD

**Issue by dspreitz** [Turbo87/aeroscore#81]
It would be nice, if aerscore could provide some key information about the task in the live-scoring view, in the line below the time , but above the table:

For Racing task:
- Type of task
- Length of task

For AAT task:
- Type of task
- distance min
- distance max
- AAT min time

This would help understand, if the relevant task parameters were correctly understood by aeroscore (and correctly provided in the task file) and provide the user with some basic information about the task.

**Comment by Turbo87** [comment]
I agree, though instead of investing time into the aeroscore CLI example I would rather put that information into the ogn-web-viewer.
